source/apps/crawl_space/views.py

```diff
@@ -296,6 +284,10 @@ def get_context_data(self, **kwargs):
             plots = AcheDashboard(self.get_object()).get_plots()
             context['scripts'] = plots['scripts']
             context['divs'] = plots['divs']
+        elif self.get_object().crawler == "nutch":
+            plots = NutchDashboard(self.get_object()).get_plots()
```

Comment:
@ahmadia are you saying that this is a good place for this? Cant understand the double-negative :D